doc/users/project/mission.rst

```diff
@@ -0,0 +1,28 @@
+Matplotlib Mission Statement
+============================
+
+The Matplotlib developer community develops, maintains, and supports Matplotlib
+and its extensions to provide plotting tools for the Scientific Python
+Ecosystem.  We judge our success based on how effectively we enable our users, across
+scientific, industrial, commercial, and educational domains, to achieve their
+goals.
+
+We believe that a diverse developer community creates the best software, and we
+welcome anyone who shares our mission and values, as described in the code of
+conduct.
+
+
+Adapting the requirements :ref:`laid out by John Hunter
+<project_history>` Matplotlib should:
+
+* Support all users of the Scientific Python Ecosystem;
+* Produce high-quality raster and vector format outputs;
```

Comment:
Sorry for dropping in late in the discussion (thanks to @story645 for pointing me to it as well). Perhaps it is worth mentioning "publication quality plots" in there as well?